### PR TITLE
Fb toolbar

### PR DESCRIPTION
**Pull request by JiajunSang**
Created new toolbar function on the top, and left side navigation drawer function.
Modified the page layout for each activity, especially main activity.
Did not convert each activity to fragment, but I added toolbar on each of them to make them look consistent.

### CODE REVIEW

File: app/src/main/java/ch/epfl/sdp/MainActivity.java [hunk 1]
```diff
@@ -2,57 +2,151 @@
...
-        });
+//        // timetable button
+//        Button timetableBtn = (Button)findViewById(R.id.timetableBtn);
+//        timetableBtn.setVisibility(View.INVISIBLE);
+//        timetableBtn.setOnClickListener(new View.OnClickListener() {
+//            @Override
+//            public void onClick(View v) {
+//                Intent startIntent = new Intent(getApplicationContext(), Timetable.class);
+//                startActivity(startIntent);
+//            }
```
Comment (realXinyangLu) [inline app/src/main/java/ch/epfl/sdp/MainActivity.java]: Are these comments neccesary?

File: app/src/androidTest/java/ch/epfl/sdp/MainActivityTest.java [hunk 2]
```diff
@@ -15,65 +12,71 @@
...
-        onView(withId(R.id.mainloginBtn)).perform(click());
-        Activity mainloginActivity = getInstrumentation().waitForMonitorWithTimeout(monitor_login, 5000);
-        assertNotNull(mainloginActivity);
-        Espresso.pressBack();
-    }
+//    @Test
+//    public void testButtons(){
+//        //test timetable Button
+//        assertNotNull(mActivityRule.getActivity().findViewById(R.id.timetableBtn));
+//        onView(withId(R.id.timetableBtn)).perform(click());
```
Comment (realXinyangLu) [inline app/src/androidTest/java/ch/epfl/sdp/MainActivityTest.java]: Are they necessary? Maybe can delete

Comment (JiajunSang) [reply app/src/androidTest/java/ch/epfl/sdp/MainActivityTest.java]: yup i agree

**Review by realXinyangLu — APPROVED** [review]

**Review by yutong0330 — APPROVED** [review]

**Comment by JiajunSang** [comment]
travis stuck in "continuous integration", I'll open a new one to try

**Pull request closed without merging**